tests/raftstore/cluster.rs

```diff
@@ -318,18 +322,27 @@ impl<T: Simulator> Cluster<T> {
             let mut region = self.get_region(key);
             let region_id = region.get_id();
             let req = new_request(region_id, region.take_region_epoch().clone(), reqs.clone());
-            let resp = self.call_command_on_leader(region_id, req, timeout).unwrap();
-            if resp.get_header().has_error() {
-                if self.refresh_leader_if_needed(&resp, region_id) {
-                    warn!("seems leader changed, let's retry");
-                    continue;
-                } else if try_cnt == 1 && resp.get_header().get_error().has_stale_epoch() {
-                    warn!("seems split, let's retry");
-                    try_cnt += 1;
+            let result = self.call_command_on_leader(region_id, req, timeout);
+            match result {
+                Ok(resp) => {
+                    if resp.get_header().has_error() {
+                        if self.refresh_leader_if_needed(&resp, region_id) {
+                            warn!("seems leader changed, let's retry");
+                            continue;
+                        } else if try_cnt <= 5 && resp.get_header().get_error().has_stale_epoch() {
+                            warn!("seems split, let's retry");
+                            try_cnt += 1;
+                            continue;
+                        }
+                    }
+                    return resp;
+                }
+                Err(Error::Timeout(_)) => {
+                    warn!("call command timeout, let's retry");
```

Comment:
It's two kind of thing:
one is we know exactly that timeout reason caused by "current cluster has no leader" or "we send request to stale leader, which don't reply because it's not leader any more "
the other is we don't know what cause timeout but we have a idempotent operation, it's okay to retry.

retry the former case fix CI but not always right (if timeout by other reason)
retry the later case always do things right but never fix anything